sdk/javascript/client/deferred.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Intel Corporation
+ * Copyright 2017 Intel Corporation
```

Comment:
I believe the correct header for files made in 2016, but updated in 2017 is _"Copyright 2016, 2017"_